.travis.yml

```diff
@@ -63,6 +63,9 @@ jobs:
     name: "Compiling"
     script:
     - mkdir -p ~/.local/bin
+    - travis_retry curl -L -o cardano-node-simple-3.0.1.tar.gz https://raw.githubusercontent.com/input-output-hk/cardano-wallet/master/test/data/cardano-node-simple/cardano-node-simple-3.0.1.tar.gz
+    - tar xzf cardano-node-simple-3.0.1.tar.gz -C $HOME/.local/bin
+    - cardano-node-simple --help
```

Comment:
Will remove this extra one on rebase after #131 is merged, this is mainly for testing and checking that things works well in CI.